content/docs/getting-started/qri-cli-quickstart.md

```diff
@@ -48,7 +48,7 @@ $ qri registry prove --username foo --password applesauce
 While it’s possible to use Qri without an account, signing up ensures that your username is unique and gives you access to Qri Cloud for one-click data publishing.  Choose a good username, it will be used to reference each of your datasets on the Qri network.  
 
 ```
-$ qri registry signup --username foo --email foo@qri.io --password applesauce
+$ qri registry prove --username foo --email foo@qri.io
```

Comment:
`signup` is correct here. `prove` is a different thing where an existing user links a new private key but doesn't create a fresh account. The field updates are correct.

Reply:
Sigh.

Redid the edit because I had other commits in the history. And changed the wrong line.

Fixing.